app/Http/Integrations/Requests/GetWorkflowJobsRequest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Integrations\Requests;
+
+use App\DTO\WorkflowJobDTO;
+use App\DTO\WorkflowRunDTO;
+use App\Models\Organization;
+use App\Models\Repository;
+use App\Models\WorkflowRun;
+use App\Services\CalculateJobTimeService;
+use App\Services\GetRunnerDataService;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Auth;
+use Saloon\Enums\Method;
+use Saloon\Http\Request;
+use Saloon\Http\Response;
+
+class GetWorkflowJobsRequest extends Request
+{
+    protected Method $method = Method::GET;
+
+    public function __construct(
+        protected WorkflowRunDTO $workflowRunDto,
+    ) {}
+
+    public function resolveEndpoint(): string
+    {
+        $repository = Repository::query()
+            ->where("id", $this->workflowRunDto->repositoryId)
+            ->firstOrFail();
+
+        $organization = Organization::query()
+            ->where("id", $repository->organization_id)
+            ->firstOrFail();
```

Comment:
Here, we can pass oragnization and repository ids as parameters.
`firstOrFail` actions we can do in controller ot smth liek that.

Reply:
now passing organization and repository name as parameters